.github/workflows/tests.yml

```
@@ -6,19 +6,14 @@ jobs:
   build_ubuntu:
     strategy:
       matrix:
-        python:
-          - '3.7'
-          - '3.8'
-          - '3.9'
-          - '3.10'
-          - '3.11'
-    name: ${{ matrix.python }}
+        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12-dev"]
```

Was 3.7 dropped deliberately?

```suggestion
        python-version: ["3.7", "3.8", "3.9", "3.10", "3.11", "3.12-dev"]
```

---

Good catch, that was accidental :)